apps/web/app/[locale]/timesheet/[memberId]/components/CalendarView.tsx

```
@@ -1,7 +1,142 @@
+                                                    task={task.task}
+                                                    className={clsxm(
+                                                        'shadow-[0px_0px_15px_0px_#e2e8f0] dark:shadow-transparent'
+                                                    )}
+                                                    taskTitleClassName={clsxm(
+                                                        'text-sm text-ellipsis overflow-hidden !text-[#293241] dark:!text-white '
+                                                    )}
+                                                    showSize={true}
+                                                    dash
+                                                    taskNumberClassName="text-sm"
+                                                />
+                                                <span className="flex-1">{task.project && task.project.name}</span>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)